buildbot-master.sls

```diff
@@ -65,3 +65,74 @@ buildbot-github-listener:
     - watch_in:
       - service: buildbot-github-listener
 
+iptables:
+  service.running:
+    - enable: True
+    - reload: True
+
+# Open TCP ports for nginx, Homu, Buildbot, ssh
+{% for port in '54856','9001','9010','ssh' %}
+iptables:
+  iptables.append:
+    - table: filter
+    - chain: INPUT
+    - jump: ACCEPT
+    - dport: ssh
+    - proto: tcp
+    - save: True
+{% endfor %}
+
+# Enable SSH IPv6 connections
+iptables:
+  iptables.append:
+    - table: filter
+    - chain: INPUT
+    - jump: ACCEPT
+    - dport: ssh
+    - proto: tcp
+    - save: True
+    - family: ipv6
+
+# Open ports for ntp
+{% for family in 'ipv4','ipv6' %}
+iptables:
+  iptables.append:
+    - table: filter
+    - chain: INPUT
+    - jump: ACCEPT
+    - dport: ntp
+    - proto: udp
+    - save: True
+    - family: {{ family }}
+{% endfor %}
+
+# Open the ports Salt needs, but only to the minions.
+# Minion IPs should probably come from Pillar? Spelling out the list here
+# doesn't seem like the right way.
+{% for minion in '208.52.161.130', '208.52.161.128', '63.135.170.19',
+    '208.52.170.250', '66.228.48.56', '173.255.201.95', '45.79.167.177',
+    '72.14.176.110','96.126.114.185', '0.0.0.0'  %}
```

Comment:
Just an FYI here, but having '0.0.0.0' in the list I would suspect has an *ANY* effect, which makes the other IPs redundant.